tests/test_summary.py

```diff
@@ -795,6 +795,18 @@ def test_unique_datetime_for_short_timesteps(filepath):
     assert summary.df(EclFiles(filepath)).index.is_unique
 
 
+@pytest.mark.parametrize(
+    "filepath",
+    [
+        SHORT_STEP_WITH_TIMESTEP,
+        SHORT_STEP_WITH_TIMESTEP_LONG,
+    ],
+)
+def test_unique_datetime_retain_index_name(filepath):
+    """Test _ensure_unique_datetime_index method retain index name"""
+    assert summary.df(EclFiles(filepath)).index.name is not None
```

Comment:
The index should be names DATE in all scenarios I think, so maybe better to check that than not None?
```suggestion
    assert summary.df(EclFiles(filepath)).index.name == "DATE"
```

Reply:
I see. I was not sure if it's always DATE or it could be TIME for example. 

I'll update it :+1: